crates/lib/src/qpu/api.rs

```diff
@@ -129,11 +137,7 @@ pub async fn retrieve_results(
     client: &Qcs,
 ) -> Result<ControllerJobExecutionResult, GrpcClientError> {
     #[cfg(feature = "tracing")]
-    tracing::debug!(
-        "retrieving job results for {} on {}",
-        job_id,
-        quantum_processor_id
-    );
+    tracing::debug!("retrieving job results for {} on {}", job_id, job_target,);
```

Comment:
Should we be using the fields here (`%job_id`) rather than formatting?

Reply:
Depends on what we want, really. Fields allow us to filter traces by those fields if we, say, output traces as JSON. Otherwise, I think this is more readable for humans.